tests/test-entry.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -xe
 
-TEST="testcafe --sf chromium:headless  --no-sandbox --disable-dev-shm-usage "
+TEST="testcafe --sf --assertion-timeout 15000 --skip-js-errors chromium:headless --no-sandbox --disable-dev-shm-usage"
```

Comment:
Worth noting, that increasing the assertion time is not the ideal. But after lots of testing locally I found that this gave me the most stability running the browser test container. I've spent a good day on this, and my only conclusion is that the testing container testcafe/testcafe is extremely unreliable in particular on OSX. Often it would fail to even connect to the browser instance which itself had spawned in the container, and others tests just seem to stall as either the content has failed to load or things are just so slow it hasn't completed rendering the desired page. Reviewer being a SPA also compounds this.
I also felt for consistency though that the arguments should be consistent across all runs, even if in some cases they shouldn't be needed. I did try using a config file, but seems testcafe does not support mixing config files and cli args failing to honour the values in the config file. As such, this will do.
Again, I think the long term goal here should be to try and migrate the tests over to another system, e.g. Playwright, as TestCafe is just too 'flaky' to be relied on here.